src/common/transformations/src/transformations/op_conversions/detection_output_upgrade.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/detection_output_upgrade.hpp"
+
+#include <ngraph/op/util/detection_output_base.hpp>
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+
+#include "itt.hpp"
+
+using namespace std;
+using namespace ngraph;
+using namespace ngraph::op::util;
+
+NGRAPH_RTTI_DEFINITION(pass::ConvertDetectionOutput1ToDetectionOutput8, "ConvertDetectionOutput1ToDetectionOutput8", 0);
+
+pass::ConvertDetectionOutput1ToDetectionOutput8::ConvertDetectionOutput1ToDetectionOutput8() {
```

Comment:
Do we have a plugin which need to have this transformation?

Reply:
Yes, very soon. it is wanted by the CPU plugin so let us have it